docs/install/update-pro.md

```diff
@@ -8,85 +8,116 @@ This document provides instructions how you can upgrade from Percona Server for
 
 ## Procedure
 
-1. Stop the `mongod` service
+=== ":material-debian: On Debian and Ubuntu"
 
-    ```{.bash data-prompt="$"}
-    $ sudo systemctl stop mongod
-    ```
+    1. Stop the `mongod` service    
 
-2. Configure the repository
+        ```{.bash data-prompt="$"}
+        $ sudo systemctl stop mongod
+        ```    
 
-    === ":material-debian: On Debian and Ubuntu"
+    2. [Install `percona-release` :octicons-link-external-16:](https://docs.percona.com/percona-software-repositories/installing.html#__tabbed_1_1). If you have installed it before, [upgrade :octicons-link-external-16:](https://docs.percona.com/percona-software-repositories/updating.html#__tabbed_1_1) it to the latest version
 
-        1. Create the `/etc/apt/sources.list.d/psmdb-pro.list` configuration file with the following contents
+    3. Enable the repository. Choose your preferable method:            
 
-            ```ini title="/etc/apt/sources.list.d/psmdb-pro.list"
-            deb http://repo.percona.com/private/[TOKENID]-[TOKEN]/psmdb-60-pro/apt/ OPERATING_SYSTEM main
-            ```
+        === ":material-console: Command line"    
 
-        2. Update the local cache
+            Run the following command and pass your credentials to the Pro repository:            
 
             ```{.bash .data-prompt="$"}
-            $ sudo apt update
-            ```
-
-    === ":material-redhat: On RHEL and derivatives"
-
-        Create the `/etc/yum.repos.d/psmdb-pro.repo` configuration file with the following contents
+            $ sudo percona-release enable psmdb-60-pro release --user_name=<Your Customer ID> --repo_token=<Your PRO repository token>
+            ```            
 
-        ```ini title="/etc/yum.repos.d/psmdb-pro.repo"
-        [psmdb-6.0-pro]
-        name=PSMDB_6.0_PRO
-        baseurl=http://repo.percona.com/private/[TOKENID]-[TOKEN]/psmdb-60-pro/yum/main/$releasever/RPMS/x86_64
-        enabled=1
-        gpgkey = https://repo.percona.com/yum/PERCONA-PACKAGING-KEY
-        ```
+        === ":octicons-file-code-24: Configuration file"            
 
-3. Install Percona Server for MongoDB Pro packages
+            Create the `/root/.percona-private-repos.config` configuration file with the following content:            
 
-    === ":material-debian: On Debian and Ubuntu"
+            ```ini title="/root/.percona-private-repos.config"
+            [psmdb-60-pro]
+            USER_NAME=<Your Customer ID>
+            REPO_TOKEN=<Your PRO repository token>
+            ```            
```

Comment:
We'd still need to execute command like 
sudo percona-release enable psmdb-60-pro release

If we even want to keep this sample

Reply:
Updated. As discussed with @radoslawszulgo we keep both options in the docs